### ISSUE TITLE
Icon's size decresased after install

### ISSUE THREAD

**Issue by somepage** [micheleg/dash-to-dock#507]
After installing of extension icon's size in "Show applications" decreased. I couldn't change this  using extension settings.
Before
![image](https://cloud.githubusercontent.com/assets/16035955/26522911/b8bd51e4-4313-11e7-8206-c8273603581b.png)

After
![image](https://cloud.githubusercontent.com/assets/16035955/26522908/8fca9a44-4313-11e7-8b30-caff9d666051.png)

**Comment by micheleg** [comment]
Where did you position the dock? it looks like some space is reserved for a bottom dock. If the dock is at the bottom than that makes sense as the app grid is scaled down? I don't see the dock on the second screenshot: if the dock is on a secondary monitor, than it could be this issue #410 ?

**Comment by somepage** [comment]
Yes, it's located to bottom. I understood this. But is dash displayed over menu of applications?

**Comment by micheleg** [comment]
I'm not sure I understand your question. I would expect to see the dock on the above screenshot. Are you using other extensions?
